### PR TITLE
make various inputs conform to style guide

### PR DESCRIPTION
**Pull request by panentheos**
Notable changes:
* universal border-box sizing
* uniform height buttons and text boxes (2.25rem = 36px)

### CODE REVIEW

File: src/components/common.js [hunk 1]
```diff
@@ -81,8 +82,8 @@ export const textInput = function(props) {
   return input(_.merge({
     style: _.merge({
       width: '100%',
-      padding: '0.5rem 1rem',
-      fontWeight: 300, fontSize: '1em'
+      paddingLeft: '1rem', paddingRight: '1rem',
+      fontWeight: 300, fontSize: 14
```
Comment (zarsky-broad) [inline src/components/common.js:86]: Legit

File: src/components/icons.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const icon = function(shape, props) {
 }
 
 export const breadcrumb = function(props) {
-  return icon('angle right', _.merge({ size: 10, style: { padding: '0 0.25rem' } }, props))
+  return icon('angle right', _.merge({ size: 10, style: { margin: '0 0.25rem' } }, props))
```
Comment (zarsky-broad) [inline src/components/icons.js:22]: 👍

File: src/pages/workspaces/workspace/Container.js [hunk 3]
```diff
@@ -113,7 +114,7 @@ export class WorkspaceContainer extends Component {
       contextBar({
         style: {
           paddingLeft: '5rem', borderBottom: `5px solid ${Style.colors.secondary}`,
-          textAlign: 'center', color: 'white', lineHeight: '3.5rem', textTransform: 'uppercase'
+          textAlign: 'center', color: 'white', lineHeight: '3.75rem', textTransform: 'uppercase'
```
Comment (zarsky-broad) [inline src/pages/workspaces/workspace/Container.js:117]: The only issue here is that the options in the side nav no longer align with the bar height, so you should tweak those too.

**Review by zarsky-broad — APPROVED** [review]

**Comment by zarsky-broad** [comment]
I super appreciate all the simplification.

**Review by MattPutnam — APPROVED** [review]

File: src/pages/workspaces/workspace/Container.js (hunk 3 above)
Comment (panentheos) [reply src/pages/workspaces/workspace/Container.js:117]: I didn't realize that was deliberate. I'll patch it.

**Pull request merged**